docs/source/data-proc-additional.ipynb

```diff
@@ -54,7 +54,68 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "<!-- ### ADD FIGURE -->"
+        "(data-proc:z-variables)=\n",
+        "## Vertical coordinate (_z_-axis) variables"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The SONAR-netCDF4 v1 convention defines several variables in the `Platform` group that are referenced to a common Platform Coordinate System (PCS) vertical (_z_-axis) origin, ${PCS}_o$. These variables, when available, enable the calculation of the depth of each echo sample from the water surface. These variables are listed below together with their definitions, with edits when appropriate."
```

Comment:
In my original text, I had this additional statement: "as well for individual instruments". That was to account for `MRU_offset_z` and `position_offset_z`, which are not necessary for the calculation of the depth of each echo sample from the water surface. I included them in the list for completeness, but deliberately excluded them from my figure sketch to focus the figure on variables used in calculating `sample_depth`. Do you think it's fine to leave this text as is, or should we bring back a statement that clarifies that not all the variables in the list are necessary to calculate `sample_depth`?

Reply:
I think it is ok to leave it as is, since in theory people should be using `add_depth` to calculate depth? Speaking of that -- do you want to add a statement about it? I forgot where that issue stands at the moment. If not finished, it'd be good to point to it as issue in the working.

Reply:
The new section being added in this PR plays double duty. On the one hand, it's in the Data Processing part of the documents, where the role of `consolidate.add_depth` is central. If it were solely focused on that, I'd suggest we drop `MRU_offset_z` and `position_offset_z` altogether since they don't come into play and are a distraction. But, at least as I intended it, it's also partly a summary of the convention's and echopype's storage of vertical-dimension data in a converted EchoData object.

> Speaking of that (`add_depth`) -- do you want to add a statement about it? I forgot where that issue stands at the moment. If not finished, it'd be good to point to it as issue in the working.

We pushed the issue (#790) to 0.8.1. I intend it to be my first priority after we release 0.8.0.

I can mention `add_depth` in this section, but only briefly, mentioning that it'll be more robust soon.

Reply:
In the latest commit I've added a brief mention of `add_depth` in an admonition box, including a link to the open issue and upcoming improvements. 

I did not change anything vis a vis `MRU_offset_z` and `position_offset_z`

Reply:
Sounds good. Thanks!